db.go

```diff
@@ -203,6 +206,14 @@ func (db *db) upgrade(version *version) {
 	db.mux.upgrade(version)
 	version.setState(versionActive)
 
+	if version.stats != nil {
+		title := fmt.Sprintf("A new version is active in %s.", db.name)
+		text := fmt.Sprintf("Version %s has been set to active in db %s.", version.name, db.name)
+		event := statsd.NewEvent(title, text)
```

Comment:
Does this honor the config to override the statsd url?

Reply:
It should, the url is set when [the client is created](https://github.com/stripe/sequins/blob/56fc7e96de1dee46ce50a3d70aedb7680b96c102/sequins.go#L73-L81). I'll also just manually confirmed

Reply:
Whoops meant to say that I did manually confirm.